src/latex2sympy/latex2antlrJson.cpp

```diff
@@ -0,0 +1,294 @@
+#include <chrono>
+#include <sstream>
+#include "antlr4-runtime.h"
+#include <json/json.h>
+#include <pybind11/pybind11.h>
+#include "LATEXLexer.h"
+#include "LATEXParser.h"
+
+using namespace latex2antlr;
+using namespace antlr4;
+using namespace std::chrono;
+
+/// @brief Return the friendly name for the given ParseTree if it is a RuleContext.
+std::string getRuleName(tree::ParseTree *tree, LATEXParser *parser) {
```

Comment:
you could add `using namespace std::string`, `using namespace std::vector;` etc. and not have to do all these std:: things. It's fine either way though.

Reply:
@lily514 thanks took a crack at some cleanup, seems pretty good now